src/app/base/components/header/header.component.html

```diff
@@ -152,6 +152,13 @@
                         }" (optionClicked)="executeOption($event)"></kern-dropdown>
                 </div>
             </div>
+            <div class="flex justify-center overflow-visible" *ngIf="hideLogout">
```

Comment:
not sure if this makes sense regarding the naming of the variable "hideLogout" -> if it's true we give a logout? 

i'd go without adding a new block but changing the existing one with currently *ngIf="!hideLogout"


optionArray: isDemo? ['Logout']: showConfigSettings ? ['Account Settings', 'Change config', 'Logout'] : ['Account Settings', 'Logout'],

and create a variable isDemo to access and change the hidelogout to work with a different condition (currently fixed to false)